.github/workflows/build-and-release.yml

```
@@ -134,37 +134,176 @@ jobs:
           name: PictoPy-MacOS
           path: backend/dist/PictoPy_Server/PictoPy-MacOS.zip
 
+  build-sync-microservice-windows:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)